sqlite-android/build.gradle

```diff
@@ -132,25 +141,25 @@ publishing {
             artifact "build/outputs/aar/${project.name}-release.aar"
             artifact sourceJar
             artifact javadocJar
-            pom {
-                name = project.name
-                description = project.description
-                url = 'https://github.com/requery/sqlite-android'
-                licenses {
-                    license {
-                        name = 'The Apache License, Version 2.0'
-                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-                    }
-                }
-                scm {
-                    connection = 'scm:git:git://github.com/requery/sqlite-android.git'
-                    url = 'https://github.com/requery/sqlite-android'
-                }
-            }
+//            pom {
+//                name = project.name
+//                description = project.description
+//                url = 'https://github.com/requery/sqlite-android'
+//                licenses {
+//                    license {
+//                        name = 'The Apache License, Version 2.0'
+//                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+//                    }
+//                }
+//                scm {
+//                    connection = 'scm:git:git://github.com/requery/sqlite-android.git'
+//                    url = 'https://github.com/requery/sqlite-android'
+//                }
+//            }
             pom.withXml {
                 asNode().children().last() + project.pomXml
                 def dependencies = asNode().appendNode('dependencies')
-                configurations.compile.allDependencies.each {
+                configurations.compile.allDependencies.all {
```

Comment:
gradle7 compatibility - this has to do with when things are evaluated and all the "deferred configuration" stuff they are doing for performance. No semantic difference otherwise